Gemfile

```diff
@@ -1,16 +1,15 @@
 source 'https://rubygems.org'
 
 gem 'inifile'
-gem 'azure_mgmt_compute', '= 0.14.0'
-gem 'azure_mgmt_resources', '= 0.14.0'
-gem 'azure_mgmt_network', '= 0.14.0'
+gem 'azure_mgmt_resources', '~> 0.15'
 gem 'bundle'
 
 group :development do
   gem 'rake'
   gem 'rubocop'
   gem 'github_changelog_generator'
   gem 'pry-coolline'
+  # gem 'pry-byebug'
```

Comment:
I would remove this line.

Reply:
I can remove this line, however does it make a difference that it is in there?
It makes debugging things easier during development.

Reply:
It does, because there's no explanation to the next user as to why it's here and why it's commented out. And whether it's commented out or if you add it later, you're still modifying the file.

If we want to consider adding `pry-byebug` as part of our standard Gemfile inclusion for both inspec-aws and inspec itself, let's address that as a separate PR to people can chime in and have their voices heard.

Reply:
_embarassed_

I apologsie for this. I did not reember / realise / spot that I had commented this out. I have removed it now in the latest push.